PSI_Project/Controllers/GoalsController.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Microsoft.AspNetCore.Mvc;
```

Comment:
please lookup how other controllers were implemented and change yours accordingly, we need to write unified code! Same goes with other classes.

Reply:
Changed all controllers with constructor injection (DI) and modified program.cs file to provide the dependencies, so now all the controllers look like goalsController.